### ISSUE TITLE
Python reports wrong include paths

### ISSUE THREAD

**Issue by extrowerk** [haikuports/haikuports#6543]
```
~> python3.8
Python 3.8.12 (default, Nov 20 2021, 23:55:23) 
[GCC 11.2.0] on haiku1
Type "help", "copyright", "credits" or "license" for more information.
>>> import sysconfig
>>> from pprint import pprint
>>> pprint(sysconfig.get_paths())
{'data': '/packages/python38-3.8.12-1/.self',
 'include': '/packages/python38-3.8.12-1/.self/include/python3.8',
 'platinclude': '/packages/python38-3.8.12-1/.self/include/python3.8',
 'platlib': '/packages/python38-3.8.12-1/.self/lib/python3.8/site-packages',
 'platstdlib': '/packages/python38-3.8.12-1/.self/lib/python3.8',
 'purelib': '/packages/python38-3.8.12-1/.self/lib/python3.8/site-packages',
 'scripts': '/packages/python38-3.8.12-1/.self/bin',
 'stdlib': '/packages/python38-3.8.12-1/.self/lib/python3.8'}
>>> 
```

Notice the inclode folder in each path.

See this bug for more info: https://bugs.launchpad.net/calibre/+bug/1958083

**Comment by eli-schwartz** [comment]
As mentioned in the calibre bug: https://github.com/haikuports/haikuports/blob/7bc183d2297e8fab85e2e710ea888eec0099eb87/dev-lang/python/patches/python310-3.10.1.patchset#L210-L237

`sysconfig.get_paths()` is wrong, but `sysconfig.get_paths('haiku')` will be correct. The problem is that the default when no explicit scheme is given, is supposed to be the current platform's native paths, so you will need to make sure that `get_preferred_schemes('prefix')` and `get_default_scheme()` correctly report haiku as the best scheme to use.

**Comment by korli** [comment]
I have pushed an update for python 3.9, will do for 3.8.

**Comment by korli** [comment]
fixed in 01a50ad2782cc467154e32adc9ff66536dcccd4a and before.
